presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt

```diff
@@ -1,36 +1,68 @@
 #$ TITLE: SupaflyFPV Freestyle 5 Inch
-#$ FIRMWARE_VERSION: 4.3
+#$ FIRMWARE_VERSION: 4.3, 4.4
 #$ CATEGORY: TUNE
 #$ STATUS: OFFICIAL
 #$ KEYWORDS: Supafly, SupaflyFPV, 4S, 6S, Freestyle, 5 inch, 5"
 #$ AUTHOR: SupaflyFPV
-#$ DESCRIPTION: SupaflyFPV Community Freestyle Preset for 5 Inch Builds
+
+#$ PARSER: MARKED
+
+#$ DESCRIPTION: [<img src="https://supaflyfpv.com/wp-content/uploads/2022/06/Logo-Cropped192.svg" width="220px" style="margin-left: auto; margin-right: auto; display: block;"/>](https://supaflyfpv.com)
```

Comment:
to have a link to the image you should do something like that:
https://api.betaflight.com/firmware-presets/presets/4.3/tune/UAV_tech_6-7in.txt

Reply:
ok...on recommendation - I copied the formatting as used by:

https://github.com/betaflight/firmware-presets/blob/master/presets/4.3/rates/mouseFreestyle.txt

Reply:
in mousefpv preset there is no link on the image.

Reply:
ok re link image change.
 re BR - I put the spaces in (like MouseFPV) in order to space the text. Left 5" is without, 6" is with:

<img width="1261" alt="Screenshot 2022-07-01 at 23 10 26" src="https://user-images.githubusercontent.com/46397720/176973389-2dab025f-bc08-466b-876c-b77ea0cb7d98.png">

Reply:
@SupaflyFPV  send me your branch name i will check with configurator + chrome dev tools

Reply:
https://github.com/SupaflyFPV/firmware-presets/tree/Fresher-Preset-Updates

Reply:
I am so confused why this one works:

`[<img src="https://supaflyfpv.com/wp-content/uploads/2022/06/Logo-Cropped192.svg" width="220px" style="margin-left: auto; margin-right: auto; display: block;"/>](https://supaflyfpv.com)`

and confused why <br> needed. I guess MARKDOWN eats empy lines.

So i guess yes, you need <br/> if you need extra spacing.

Plz make it <br/> tho. 

Sorry for the confusion